src/services/kpis/kpis.svc.coffee

```diff
@@ -385,7 +385,7 @@ angular
       promises = []
 
       for alert in alertsToCreate
-        promises.push ImpacAlerts.create(kpi.id, { alert: _.pick(alert, ['service']) })
+        promises.push ImpacAlerts.create(kpi.id, { alert: _.pick(alert, ['service']), recipients: alert.recipients })
```

Comment:
@adamaziz15 an in-app alert wont have key `recipients`, maybe do a merge if service is email instead.